pychromecast/socket_client.py

```diff
@@ -281,7 +301,7 @@ def send_message(self, destination_id, namespace, data,
         msg.destination_id = destination_id
         msg.payload_type = cast_channel_pb2.CastMessage.STRING
         msg.namespace = namespace
-        msg.payload_utf8 = json.dumps(data, ensure_ascii=False).encode("utf8")
+        msg.payload_utf8 = json.dumps(data)
```

Comment:
This breaks compatibility with Python 3.

Although Python 3 support is currently broken, it is the plan to support it as soon as Google releases version 3 of protobuf (which includes Python 3 support).